src/js/util.js

```diff
@@ -250,6 +250,33 @@ function deprecate(fn, msg, code) {
   return deprecated;
 }
 
+var customPromisifySymbol = 'util:promisify:custom';
```

Comment:
This looks like the symbol, so i think it's better to flag a TODO or FIXME as the following:

```js
// FIXME(legendecas): change to use formal `Symbol` when it is ready.
```